pkg/fanal/analyzer/pkg/dpkg/copyright_test.go

```diff
@@ -0,0 +1,150 @@
+package dpkg
+
+import (
+	"context"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/aquasecurity/trivy/pkg/fanal/analyzer"
+	"github.com/aquasecurity/trivy/pkg/fanal/types"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestDpkgLicensesAnalyzer_Analyze(t *testing.T) {
+	tests := []struct {
+		name              string
+		copyrightFilePath string
+		wantLicense       *analyzer.AnalysisResult
+	}{
+		{
+			name:              "happy path. There are 'License:' pattern and licenseclassifier",
+			copyrightFilePath: "testdata/copyrightFiles/usr/share/doc/zlib1g/copyright",
```

Comment:
Can we rename and move these files?

```suggestion
			copyrightFilePath: "testdata/machine-readable-copyright",
```

I named it based on these scripts.
https://github.com/daald/dpkg-licenses/tree/master/lib

Reply:
moved